packages/react-components/react-progress/src/utils/clampMax.test.ts

```diff
@@ -0,0 +1,44 @@
+import { clampMax } from './clampMax';
+
+describe('ProgressBar clampMax util', () => {
+  const originalConsoleError = console.error;
+  beforeEach(() => {
+    console.error = jest.fn();
+  });
+  afterAll(() => {
+    console.error = originalConsoleError;
+  });
+  it('sends an error message when max is negative', () => {
+    const max = -1;
+    const errorMsg = `The prop 'max' must be greater than 0. Received max: ${max}`;
+    clampMax(max);
+    expect(console.error).toHaveBeenCalledWith(errorMsg);
+  });
+  it('sets max to the proper value when max is negative', () => {
+    const max = -1;
+    const testMax = clampMax(max);
+    expect(testMax).toBe(1);
+  });
+  it('sends an error message when max is zero', () => {
+    const max = 0;
+    const errorMsg = `The prop 'max' must be greater than 0. Received max: ${max}`;
+    clampMax(max);
+    expect(console.error).toHaveBeenCalledWith(errorMsg);
+  });
+  it('sets max to the proper value when max is zero', () => {
+    const max = 0;
+    const testMax = clampMax(max);
+    expect(testMax).toBe(1);
+  });
+  it('does not send an error message when max is valid', () => {
+    const max = 7;
+    const errorMsg = `The prop 'max' must be greater than 0. Received max: ${max}`;
+    clampMax(max);
+    expect(console.error).not.toHaveBeenCalledWith(errorMsg);
```

Comment:
You don't need to set an error message if `max` is valid

```suggestion
    clampMax(max);
    expect(console.error).not.toHaveBeenCalled();
```